crowdin_api/api_resources/languages/tests/test_languages_resources.py

```diff
@@ -3,7 +3,10 @@
 import pytest
 from crowdin_api.api_resources import LanguagesResource
 from crowdin_api.api_resources.enums import PatchOperation
-from crowdin_api.api_resources.languages.enums import LanguagesPatchPath, LanguageTextDirection
+from crowdin_api.api_resources.languages.enums import (
+    LanguagesPatchPath,
+    LanguageTextDirection,
+)
```

Comment:
@OmAximani0 could you please rollback all the code formatting changes that are not related to the new project ID property support? It's hard to review the PR with that amount of changes

Reply:
Probably, it would be better to squash all the commits as well